<launch>
  <include file="$(find jsk_2016_01_baxter_apc)/launch_segmentation/segmentation.launch">
    <arg name="UPPER_SHELF" value="$(find jsk_apc2016_common)/config/bin_upper_shelf.yaml"/>
    <arg name="LOWER_SHELF" value="$(find jsk_apc2016_common)/config/bin_lower_shelf.yaml"/>
    <arg name="TARGET_BIN" value="l"/>
    <arg name="INPUT_POINTS" value="/right_softkinetic_camera/depth/points"/>
    <arg name="INPUT_INFO" value="/right_softkinetic_camera/rgb/camera_info"/>
    <arg name="INPUT_IMAGE" value="/right_softkinetic_camera/rgb/image_color"/>
  </include>
</launch>
